src/UI/Root/Components/PageFrame/SettingsProvider.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { useGetEnvironmentSettings } from "@/Data/Managers/V2/Environment";
+
+export const SettingsProvider: React.FC = () => {
```

Comment:
🤔 how is this beneficial compared to using the ReactQuery directly where we need it?